src/components/Navbar/Navbar.style.js

```diff
@@ -30,8 +50,47 @@ export const LinkIcons = styled(Link)`
   position: relative;
 `
 
+export const MenuHamburguer = styled.div`
```

Comment:
I think you can simplify this by only using display: none when screen is greater than a tablet - `min-width` should do the trick